src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 Released 2022-Jul-19
 
+* Add description for metrics in README.md and refined some metrics names and
```

Comment:
i'd suggest to put the name changes here. (original name -> new name).
Adding readme is not something to be mentioned in changelog.